PathPlanning/RRTStar/rrt_star.py

```diff
@@ -58,28 +60,35 @@ def planning(self, animation=True, search_until_max_iter=True):
         """
         rrt star path planning
 
-        animation: flag for animation on or off
-        search_until_max_iter: search until max iteration for path improving or not
+        animation: flag for animation on or off search_until_max_iter: search
+        until max iteration for path improving or not
         """
 
         self.node_list = [self.start]
         for i in range(self.max_iter):
             print("Iter:", i, ", number of nodes:", len(self.node_list))
             rnd = self.get_random_node()
             nearest_ind = self.get_nearest_node_index(self.node_list, rnd)
-            new_node = self.steer(self.node_list[nearest_ind], rnd, self.expand_dis)
+            new_node = self.steer(
+                self.node_list[nearest_ind], rnd, self.expand_dis)
+            near_node = self.node_list[nearest_ind]
+            new_node.cost = near_node.cost + \
+                math.sqrt((new_node.x-near_node.x)**2 +
+                          (new_node.y-near_node.y)**2)
```

Comment:
What is for this change? Related #382?

Reply:
If you remove that line, then the code will start to produce disconnected trees. The problem is that the `cost` attribute was updated inside  `choose_parent`. If `choose_parent` returns `None`, then its value remains undefined.

Reply:
OK. I understood. However, I think `np.hypot` should be used here.